plugins/http_plugin/http_plugin.cpp

```diff
@@ -362,32 +461,26 @@ namespace eosio {
    }
 
    void http_plugin::plugin_shutdown() {
-      if(my->beast_server)
-         my->beast_server->stop_listening();
-      if(my->beast_unix_server)
-         my->beast_unix_server->stop_listening();
-
       my->plugin_state->thread_pool.stop();
 
-      my->beast_server.reset();
-      my->beast_unix_server.reset();
-
       // release http_plugin_impl_ptr shared_ptrs captured in url handlers
       my->plugin_state->url_handlers.clear();
 
       fc_ilog( logger(), "exit shutdown");
    }
 
-   void http_plugin::add_handler(const string& url, const url_handler& handler, appbase::exec_queue q, int priority, http_content_type content_type) {
-      fc_ilog( logger(), "add api url: ${c}", ("c", url) );
-      auto p = my->plugin_state->url_handlers.emplace(url, my->make_app_thread_url_handler(url, q, priority, handler, my, content_type));
-      EOS_ASSERT( p.second, chain::plugin_config_exception, "http url ${u} is not unique", ("u", url) );
+   void http_plugin::add_handler(api_entry&& entry, appbase::exec_queue q, int priority, http_content_type content_type) {
+      std::string path = entry.path;
+      fc_ilog( logger(), "add api url: ${c}", ("c", path) );
```

Comment:
Would be nice to log the category and address and port (or unix socket) for each one on startup. Doesn't need to be here but seems like we should summarize that all in one log statement.

```
info  2023-05-09T20:14:32.976 nodeos    http_plugin.cpp:474           add_handler          ] add api url: /v1/chain/get_info
info  2023-05-09T20:14:32.976 nodeos    http_plugin.cpp:474           add_handler          ] add api url: /v1/chain/get_activated_protocol_features
info  2023-05-09T20:14:32.976 nodeos    http_plugin.cpp:474           add_handler          ] add api url: /v1/chain/get_block
info  2023-05-09T20:14:32.976 nodeos    http_plugin.cpp:474           add_handler          ] add api url: /v1/chain/get_block_info
```